### PR TITLE
Add status and text UI get

### PR DESCRIPTION
**Pull request by Mr-DaveDev**

### CODE REVIEW

**Comment by tosiara** [comment]
How do I access the new "text" commands? I don't see them on UI. What is the URL to access them?

**Comment by Mr-Dave** [comment]
It is the [`webcontrol_interface`](https://rawgit.com/Motion-Project/motion/master/motion_config.html#webcontrol_interface) which provides the same interface as the previous option of [`webcontrol_html_output`](https://motion-project.github.io/motion_config.html#webcontrol_html_output).  This older option was specifically put in for the handling of control of Motion from external applications as specified in the section above the 'Alert' in this section of the [guide](https://motion-project.github.io/motion_config.html#OptDetail_Stream_Webcontrol).  

It should be noted that the actions such as `pause`, `start`, `makemovie`, `snapshot`, `restart`, `quit` and changing the configuration options are executed the same between the text and HTML as well as being the same between the previous and current interface.  So the text interface is not required.  It is just easier for programs/scripts if you want to query the configuration parms or get a status/connection since there isn't a lot of HTML to parse through.  

This PR adds the status and connection information into the HTML page along with the other information.  The status/connection can be seen on the HTML page by selecting a camera and it is next to the camera name.  Pausing or starting also invokes a page refresh which will update the information.

This PR also provides a server response back to the user for a get, status and connection sent to Motion for the text version in addition to resolving responses for a `list` request.  

Finally, comment questions at the top of the module were removed regarding the whether text interface and status/connection should be perpetuated.

File: webu.c [hunk 1]
```diff
@@ -1391,6 +1392,9 @@ static void webu_html_script_action(struct context **cnt, struct webui_ctx *webu
         "      document.getElementById('act_btn').style.display=\"none\"; \n"
         "      document.getElementById('cfg_value').value = '';\n"
         "      document.getElementById('cfg_parms').value = 'default';\n"
+        "      if ((actval == \"/detection/pause\") || (actval == \"/detection/start\")) {\n"
+        "        window.location.reload(); "
```
Comment (genius3000) [inline webu.c]: Firefox (possibly more) seem to dislike this and the action doesn't get sent to motion correctly/at all (ie. pause and start aren't pausing or starting). Verified this by commenting these 3 lines out and re-trying, with pause/start actually working again. This is probably the same or a quite similar issue to what I had with trying to close the menu after click from the JS, with the action failing to happen then.

Comment (genius3000) [reply webu.c]: I tried with a timed reload (500ms) and it worked: `setTimeout(function(){ window.location.reload(); }, 500);`  
Not the **best** solution but better than no response.

**Comment by genius3000** [comment]
I re-ran my testing, as far I can tell things are working as expected (afaik); aside from the code comment regarding `/action/(pause,start)` in HTML/CSS mode.  
Text-mode actions and detection checks/actions all did as I expected and returned proper results (**note:** i just remembered I did not test /config/set and write).  
HTML/CSS-mode status indicator is nice and just that one commented issue noted. 
Would be best for those already with scripts to give this a test-run for any issues.

File: webu.c (hunk 1 above)
Comment (Mr-Dave) [reply webu.c]: I pushed a commit to add a listener event to this and it seems to be more reliable.  I forgot these are async requests.

**Comment by tosiara** [comment]
Just to confirm with the "new" text output should I expect to have hyperlinks?
When I open http://localhost:8080, all I see is:
```
Motion 4.1.1+git20180422-45e741d Running [1] Camera
0
0
```
No hyperlinks.
Manually typing `/0/detection/connection`, `/0/detection/status` or `/0/detection/start` seems to work fine like in older version

File: webu.c (hunk 1 above)
Comment (genius3000) [reply webu.c]: Ah got ya; works good now.

**Comment by Mr-Dave** [comment]
That is correct.  The previous option [webcontrol_html_output](https://motion-project.github.io/motion_config.html#webcontrol_html_output) `off` did not have hyperlinks and there are no hyperlinks in the current text output.

**Pull request merged**